easy-digital-downloads.php

```diff
@@ -384,6 +397,33 @@ private function includes() {
 		require_once EDD_PLUGIN_DIR . 'includes/install.php';
 	}
 
+	/**
+	 * Registers REST API endpoints.
+	 *
+	 * @todo move this somewhere better
```

Comment:
I don't want to put it in a class file like that, as it won't work when we eventually add auto loading. Like right now if I put it in `Endpoint.php` or `Notifications.php`, then manually `require` those files, then it would pick up that `add_action()` and run it. But in the future when we add autoloading, it won't include the class until it's used, so I can't "trigger" the class from inside that same file.

Some point after adding autoloading (so, after 3.0), I want to set up a better system for adding hooks like this and get it out of the main plugin file.